pyproject.toml

```diff
@@ -54,7 +53,6 @@ local = ["torch", "huggingface-hub", "transformers"]
 lancedb = ["lancedb"]
 postgres = ["pgvector", "psycopg", "psycopg-binary", "psycopg2-binary", "pg8000"]
 dev = ["pytest", "black", "pre-commit", "datasets"]
-autogen = ["pyautogen"]
```

Comment:
@sarahwooders I had to remove this part (the extra) by hand since it was still present after `poetry remove pyautogen`

Reply:
yeah I just do it manually too, should be fine!